src/actions/githubActions.js

```diff
@@ -1,15 +1,11 @@
 import repositories from '../config/repositories.json';
-import requestBuilder from '../helpers/requestBuilder';
+import getIssueCountForProjects from '../provider/github';
```

Comment:
If this is going to be a facade for accessing gihub maybe we could import it as github?
Then do ```github.getIssueCount(projects)```